### PR TITLE
Snp eff

### PR DESCRIPTION
**Pull request by Bart-Ver**

### CODE REVIEW

**Comment by Bart-CER** [comment]
Can this me merged?

File: easybuild/easyconfigs/s/snpEff/snpEff-4.0.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+name = 'snpEff'
+version = '4.0'
+
+homepage = 'http://snpeff.sourceforge.net'
+description = """Genetic variant annotation and effect prediction toolbox."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['{0}_v{1}_{2}_core.zip'.format(name, *version.split('.'))]
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-4.0.eb]: we typically use `%s` style string formatting, so maybe stick with that?
also, use the provided templates

```python
sources = ['%(name)s_v%(version_major)s_%(version_minor)s_core.zip']
```

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@Bart-CER: easyconfig for `Java/1.7.0_51` is missing, maybe just bump the `Java` version to 1.7.0_76 or 1.8.0_31 (which we already have)?

**Comment by Bart-CER** [comment]
I updated the config file, bumped up the java version but to a version we have installed here, I added the java config file too.

**Comment by migueldiascosta** [comment]
@Bart-Ver, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**